requests/__init__.py

```diff
@@ -56,7 +56,7 @@
 # See https://github.com/kennethreitz/requests/issues/3213 as well.
 import ssl
 
-if ssl.OPENSSL_VERSION_INFO < (1, 0, 1):
+if hasattr(ssl, 'OPENSSL_VERSION_INFO') and getattr(ssl, 'OPENSSL_VERSION_INFO') < (1, 0, 1):
```

Comment:
@Lukasa I read that issue, and maybe I'm just dense, but I don't see where dropping `requests[security]` support for 2.6 is on the table, or maybe my understanding is wrong and that was never supported. The `OPENSSL_VERSION_INFO` attribute was added in 2.7.